designdocs/provider_registration.md

```diff
@@ -24,8 +24,6 @@ Among those questions are:
 1. How does a user become a provider?
 1. What are the attributes of a provider?
 1. How will providers be identified to other network actors?
-1. How does one stop being a provider?
-1. Can provider status be revoked from a provider?
```

Comment:
Are we not going to answer these questions?

Reply:
We will, but can't at the moment. Should I keep the questions in, and leave an asterisk - that will be answered in future versions?

Reply:
Yes, especially since we're still talking about it.  I'd rather have it in there and "TBD" or similar than to not have the question there.

Reply:
I added the questions back, and explained in 'Provider De-registration' as a future feature